src/common/paths/buckets.yml

```
@@ -57,20 +128,78 @@ post:
           $ref: "../schemas/PostBucketRequest.yml"
   responses:
     "201":
-      description: Bucket created
+      description: |
+        Success.
+        Bucket created.
       content:
         application/json:
           schema:
             $ref: "../schemas/Bucket.yml"
+          examples:
+            $ref: "../responses/example-responses/PostBucketsSuccessResponse.yml"
+    "400":
+      description: |
+        Bad request.
+      content:
+        application/json:
+          schema:
+            $ref: "../schemas/Error.yml"
+    "401":
+      $ref: "../responses/AuthorizationError.yml"
+    "403":
+      description: |
+        Forbidden.
+        Bucket quota exceeded.
```

```suggestion
        The bucket quota is exceeded.
```
Maybe provide more information about "bucket quota".

---

Will provide bucket quota information in the Limitations section